components/prism-bqn.js

```diff
@@ -0,0 +1,65 @@
+Prism.languages.bqn = {
+	'shebang': {
+		pattern: /^#![ \t]*\/.*/,
+		alias: 'important',
+		greedy: true
+	},
+	'comment': {
+		pattern: /#.*/,
+		greedy: true
+	},
+	'string-literal': {
+		pattern: /"(?:[^"]|"")*"/,
+		greedy: true,
+		alias: 'string'
+	},
+	'character-literal': {
+		pattern: /'.'/,
+		greedy: true,
+		alias: 'string'
```

Comment:
`Char` is the [correct alias](https://prismjs.com/tokens.html) here.

```suggestion
		alias: 'char'
```

Reply:
Applied this change in most recent commit!